File: Java/Arrays_and_Strings/LinearSearch.java
class Innoskrit {
	static int linearSearch(int[] arr, int element)
	{
        for(int i = 0; i < arr.length; i++) {
        	if(arr[i] == element) {
        		return i;
        	}
        }

        return -1;
	}

	public static void main(String args[])
	{
		int[] arr = {41, 15, 12, 30, 42, 52, 60, 33};
        System.out.println(linearSearch(arr, 52));
        System.out.println(linearSearch(arr, 43));
	}
}

File: Python/Arrays_and_Strings/LinearSearch.py
def linearSearch(arr, element):

	for i in range(0, len(arr)):
		if arr[i] == element:
			return i

	return -1

if __name__ == '__main__':
	arr = [41, 15, 12, 30, 42, 52, 60, 33]
	print(linearSearch(arr, 52))
	print(linearSearch(arr, 43))